big_tests/tests/mod_global_distrib_SUITE.erl

```diff
@@ -382,7 +382,7 @@ test_advertised_endpoints_override_endpoints(_Config) ->
 %% from backend and starts appropriate pool.
 test_host_refreshing(_Config) ->
     mongoose_helper:wait_until(fun() -> trees_for_connections_present() end, true,
-                               #{name => trees_for_connections_present}),
+                               #{name => trees_for_connections_present, time_left =>timer:seconds(10)}),
```

Comment:
```suggestion
                               #{name => trees_for_connections_present, time_left => timer:seconds(10)}),
```

Reply:
fixed, please merge this PR a soon as CircleCI build is green.